lib/fetch-server.js

```diff
@@ -30,41 +30,41 @@ export class FetchServer {
    * errorCallback param allows hooking test framework error handling utils
    * into FetchServer so that tests may be reliably failed on error.
    */
-  constructor(handlers, {debug = false, errorCallback = null} = {}) {
+  constructor(handlers, {debug = false, errorCallback = null, sort = false} = {}) {
     this.handlers = handlers;
     this.debug = debug;
     this.errorCallback = errorCallback;
+    this.sort = sort;
   }
 
   handle(url, params) {
     // Order routes by url length (descending).
     // This ensures that when route overlap occurs (e.g. /foo/bar and /foo/bar/baz)
     // the more-specific route (/foo/bar/baz) is checked first. Otherwise, some
     // routes will be silently ignored when more-generic (shorter) routes exist.
-    const routes = Object.entries(this.handlers).sort(([aRoute], [bRoute]) => {
-      // If route lengths are the same, sort alphabetically;
-      // otherwise, sort by route string length (descending).
-      // Route length + alpha makes sort fully deterministic.
-      if (bRoute.length === aRoute.length) {
-        return bRoute.toLowerCase() < aRoute.toLowerCase() ? 1 : -1;
-      } else {
-        return bRoute.length - aRoute.length;
-      }
-    });
+    const routes = Object.entries(this.handlers);
+    if (this.sort) {
+      routes.sort(([aRoute], [bRoute]) => {
+        // If route lengths are the same, sort alphabetically;
+        // otherwise, sort by route string length (descending).
+        // Route length + alpha makes sort fully deterministic.
+        if (bRoute.length === aRoute.length) {
+          return bRoute.toLowerCase() < aRoute.toLowerCase() ? 1 : -1;
+        } else {
+          return bRoute.length - aRoute.length;
+        }
+      });
+    }
 
     for (const [route, handler] of routes) {
       if (url.includes(route)) {
         if (params && params.body) {
-          if (isString(params.body)) {
```

Comment:
this is another revert for changes in 0.4.0 that breaks current testcases. request params.body could be object or string